src/annotator/config/settings.js

```diff
@@ -189,20 +121,20 @@ export default function settingsFrom(window_) {
   function hostPageSetting(name, options = {}) {
     const allowInBrowserExt = options.allowInBrowserExt || false;
     const hasDefaultValue = typeof options.defaultValue !== 'undefined';
-    // Optional coerce method, or a no-op.
-    const coerceValue =
-      typeof options.coerce === 'function' ? options.coerce : name => name;
```

Comment:
I think I intended it to be part of this because of all the test changes too. What I'll do is update the git commit message to explain this a bit better.